docs/ref/contrib/admin/index.txt

```diff
@@ -1746,6 +1746,18 @@ templates used by the :class:`ModelAdmin` views:
     Should return ``True`` if adding an object is permitted, ``False``
     otherwise.
 
+.. method:: ModelAdmin.has_view_permission(request, obj=None)
+
+    .. versionadded:: 1.11
+
+    Should return ``True`` if viewing obj is permitted, ``False`` otherwise.
+    If obj is ``None``, should return ``True`` or ``False`` to indicate whether
+    viewing of objects of this type is permitted in general (e.g., ``False``
+    will be interpreted as meaning that the current user is not permitted to
+    view any object of this type).
+    The ``view`` permission has been made with backward compatibility in mind, so
+    ``change`` permission imply that user can also view objects.
```

Comment:
Although the change is backwards compatible by allowing users who can _change_ to implicitly be allowed to _view_, and this makes sense because you have to see something to edit it, there is still a problem. Existing models will not have a `Permission` object for the _view_ permission.

There needs to be some documentation explaining how to add the missing permissions or perhaps a management command to create permissions objects for all models where they are found to be missing. I'll defer to @timgraham on this.